### DISCUSSION TITLE
admin password incorrect on development instance

### DISCUSSION THREAD

**Discussion by naught101** [mastodon/mastodon#22723] [General]
### Steps to reproduce the problem

1. Install a development instance using the instructions at https://docs.joinmastodon.org/dev/setup/
2. got to `http://mastodon.local`,
3. Click "sign in", and try to log in with user: `admin@mastodon.local`, password: `mastodonadmin`
4. Login fails with "Invalid E-mail address or password."

### Expected behaviour

I should be able to login

### Actual behaviour

Login fails

### Detailed description

I think the domain is correct, because it's reported on the front page. But also the admin user isn't shown on that page yet:

![image](https://user-images.githubusercontent.com/167164/208279050-5e288c02-0da7-467a-bf37-f44da50a703b.png)

It also seems odd that I'm instructed to log in with an "email", but I think that's a mastodon handle, right?

### Specifications

Mastodon: latest main branch

Running on Ubuntu 22.04.1 LTS

**Comment by naught101** [comment]
Trying to modify the account from the command line seems to indicate that it doesn't exist:

```
vagrant@mastodon:/vagrant$ bin/tootctl accounts modify admin@mastodon.local --reset-password
No user with such username
vagrant@mastodon:/vagrant$ bin/tootctl accounts modify admin --reset-password
No user with such username
```

**Comment by naught101** [comment]
I got around this by creating a new admin:

```
bin/tootctl accounts create me --email=me@gmail.com
bin/tootctl accounts modify me --confirm
bin/tootctl accounts modify me --role=Admin
```

I think maybe something like this needs to be added to the docs? Or the vagrantfile needs fixing?

**Comment by ineffyble** [comment]
Do you have the logs from when you first ran Vagrant? It should have run `bundle exec rails db:setup` which should have created `admin@mastodon.local`.

**Comment by dotupNET** [comment]
You can use the user name, not email. `bin/tootctl accounts modify admin --reset-password`.
